libs/core/src/scss/interaction-state/_index.scss

```diff
@@ -0,0 +1 @@
+@forward 'hover';
```

Comment:
We should double check this does not cause ngcc to catch fire - barrel files of various kinds gave us some problems with that mystery bug some time ago \*thousand yard stare\*